.deployignore

```diff
@@ -125,3 +125,4 @@ yarn.lock
 /wp-parsely-*/CODE_OF_CONDUCT.md
 /wp-parsely-*/init.py
 /wp-parsely-*/tests
+/wp-parsely-*/phpunit-integration.xml.dist
```

Comment:
```suggestion
/wp-parsely-*/*.dist
/wp-parsely-*/.*
/wp-parsely-*/*.json
/wp-parsely-*/webpack.config.js
```

What do you think?

Reply:
And maybe then remove `/wp-parsely-*/.prettierrc`

Reply:
I don't necessarily agree with the changes. `/wp-parsely-*/*.dist` is redundant because most of the `.dist` files are already ignored globally on the top of this file. I'd rather keep it explicit.

Reply:
I have removed the `CODE_OF_CONDUCT.md` ignore, since we don't have such file in the Parse.ly plugin.